pkg/util/pem_test.go

```diff
@@ -150,28 +160,24 @@ func TestValidateAndSanitizePEMBundle(t *testing.T) {
 			var expiredCerts []*x509.Certificate
 
 			for _, cert := range certs {
-				parsedCerts, err := DecodeX509CertificateChainBytes(cert)
-				if err != nil {
-					t.Errorf("failed to decode split PEM cert: %s", err)
-					continue
-				}
-
-				if len(parsedCerts) != 1 {
-					// shouldn't ever happen since we're decoding a single PEM cert
-					t.Errorf("got more than one parsed cert after splitting a PEM bundle")
-					continue
-				}
-
-				parsedCert := parsedCerts[0]
-
-				if parsedCert.NotAfter.Before(dummy.DummyInstant()) {
-					expiredCerts = append(expiredCerts, parsedCert)
+				if cert.NotAfter.Before(dummy.DummyInstant()) {
+					expiredCerts = append(expiredCerts, cert)
 				}
 			}
 
 			if test.expectExpiredCerts != (len(expiredCerts) > 0) {
 				t.Errorf("expectExpiredCerts=%v but got %d expired certs", test.expectExpiredCerts, len(expiredCerts))
 			}
+
+			if test.expectDuplicatesCerts {
```

Comment:
It's getting late, but shouldn't we check for duplicate certs regardless of the `expectDuplicatesCerts` value? Like we do for the expired cert test above?

Reply:
Hello,
This was done, because previous test `func TestBundlesDeduplication` in `source_test.go` has been removed. I guess it is reasonable because `source_test.go` contains tests for SOURCES, however de-duplication process takes place on the different level :)